adapter/main.go

```diff
@@ -103,6 +104,8 @@ func (a *Adapter) makeProvider(ctx context.Context, globalHTTPTimeout time.Durat
 	broadcaster := record.NewBroadcaster()
 	recorder := broadcaster.NewRecorder(scheme, corev1.EventSource{Component: "keda-metrics-adapter"})
 	handler := scaling.NewScaleHandler(kubeclient, nil, scheme, globalHTTPTimeout, recorder)
+	externalMetricsInfo := &[]provider.ExternalMetricInfo{}
```

Comment:
@zroubalik do you need to use a pointer to this slice?

Reply:
We need to modify the slice from different go routines: 
https://github.com/kedacore/keda/blob/bb97f0d9d40b37846374971e8a325415250fbf0d/controllers/keda/metrics_adapter_controller.go#L104